Documentation/ExtensionArchitecture/ComposerJson/Index.rst

```diff
@@ -253,18 +247,18 @@ Example for extension key **bootstrap_package**:
       }
    }
 
-replace
--------
+Properties no longer used
+=========================
 
-(*usually not required*)
+version
+-------
 
-`replace <https://getcomposer.org/doc/04-schema.md#replace>`__ in a
-:file:`composer.json` file specifies which other packages can be
-replaced by this package. This means that packages with different
-vendor name or package name will be treated as the same package by
-Composer.
+Was used in earlier TYPO3 versions.
```

Comment:
An additional hint that the version should properly be tagged via repo would also be a good hint. Composer takes the version information from tags in the first place if available.